htdocs/expedition/list.php

```diff
@@ -260,7 +263,7 @@
 if ($sall || $search_product_category > 0 || $search_user > 0) {
 	$sql = 'SELECT DISTINCT';
 }
-$sql .= " e.rowid, e.ref, e.ref_customer, e.date_expedition as date_expedition, e.weight, e.weight_units, e.date_delivery as delivery_date, e.fk_statut, e.billed, e.tracking_number, e.fk_shipping_method,";
+$sql .= " e.rowid, e.ref, e.ref_customer, e.date_expedition as date_expedition, e.weight, SUM(pd.subprice*ed.qty) as MontantHT, e.weight_units, e.date_delivery as delivery_date, e.fk_statut, e.billed, e.tracking_number, e.fk_shipping_method,";
```

Comment:
Not so easy.
The SUM here is not compatible with rest of sql because when 
$sall, $search_product_category or $search_user is defined, lines are duplicated and then filtered by a distinct. This make the sum counting several times the amount in line (distinct is done after the sum).
Before being able to add a join on product line grouping by a sum, we must first obtain a sql request than do not use any more the distinct.
This can be done by a PR that filter on category doing a 
`AND EXISTS (SELECT rowid from .MAIN_DB_PREFIX.categorie_societe as cs ON s.rowid = cs.fk_soc AND cp.fk_categorie = ".((int) $search_product_category))`
instead of introducing a join that duplicate record.
With this, we have no more reason to include a distinct to get the list of shipment.

For the search on user, the distinct is useless because the join does not generate duplication of lines.

For the search on sall, the distinct beome useless too because of the group by that you introduce.